.github/workflows/lint.yml

```diff
@@ -1,5 +1,21 @@
 name: Lint
 
+# https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#permissions
+permissions:
+  actions: none
+  checks: none
+  contents: none
```

Comment:
I don't know if this is needed to be set to read for commits or not. See https://docs.github.com/en/rest/reference/permissions-required-for-github-apps as there are a few settings here I'm not sure we would want to allow by setting `contents` to `read`.

Reply:
I suspect the token does not need read added because the contents of the repo itself are public.  This might bear some validation testing with private repos though.

Reply:
Works as expected when the repo is public
<img width="944" alt="image" src="https://user-images.githubusercontent.com/60357436/156235838-fc0126fa-3f18-4a17-b3f7-722ebfaf8d22.png">